erpnext/buying/doctype/request_for_quotation/request_for_quotation.py

```diff
@@ -28,6 +28,18 @@ def validate_common(self):
 		pc = frappe.get_doc('Purchase Common')
 		pc.validate_for_items(self)
 
+	def before_save(self):
+		self.update_email_id()
```

Comment:
Either put this in `validate` or change `validate` to `before_save` as both are one and the same